Application/EdFi.Ods.WebApi/Web.Npgsql.config

```diff
@@ -47,14 +50,8 @@
     </session-factory>
   </hibernate-configuration>
   <appSettings>
-    <add key="apiStartup:type" value="sharedinstance" xdt:Transform="Replace" xdt:Locator="Match(key)" />
     <add key="databaseTemplate:minimalTemplateScript" value="PostgreSQLMinimalTemplate" xdt:Transform="Replace" xdt:Locator="Match(key)" />
     <add key="databaseTemplate:populatedTemplateScript" value="PostgreSQLPopulatedTemplate" xdt:Transform="Replace" xdt:Locator="Match(key)" />
-    <add key="extensions:featureIsEnabled" value="true" xdt:Transform="Replace" xdt:Locator="Match(key)" />
-    <add key="composites:featureIsEnabled" value="false" xdt:Transform="Replace" xdt:Locator="Match(key)" />
-    <add key="profiles:featureIsEnabled" value="false" xdt:Transform="Replace" xdt:Locator="Match(key)" />
-    <add key="identityManagement:featureIsEnabled" value="false" xdt:Transform="Replace" xdt:Locator="Match(key)" />
-    <add key="ownershipBasedAuthorization:featureIsEnabled" value="false" xdt:Transform="Replace" xdt:Locator="Match(key)" />            
```

Comment:
do these all work now for postgres?

Reply:
yes, and ownership by default is disabled.

Reply:
@jamessmckay , @tbmeador-dlp we should keep profiles and identity management disabled by default. They are not pat of the LEA focused offering. composite should be enabled. 

Should be also remove so that sandbox mode is selected by default https://github.com/Ed-Fi-Alliance-OSS/Ed-Fi-ODS-Implementation/blob/development/Application/EdFi.Ods.WebApi/Web.Npgsql.config#L50

Reply:
done